adapter/pkg/logging/log_constants.go

```diff
@@ -29,3 +29,25 @@ const (
 	infoLevel       = "INFO"
 	debugLevel      = "DEBG"
 )
+
+// Log Formalization Constants
+const (
+	plainTextType = "TEXT"
+	jsonType      = "JSON"
+)
```

Comment:
```suggestion
const (
	TEXT = "TEXT"
	JSON      = "JSON"
)

That way can avoid any confusions that there is a sub json type or a sub plaintext type, when reading the code
```

Reply:
Fixed in https://github.com/wso2/product-microgateway/pull/2633/commits/1a0a7366a2d08d46530fd6a2093276f808d8e980